bot/exts/help_channels/_cog.py

```diff
@@ -642,3 +644,16 @@ async def helpdm_command(
         else:
             await _caches.help_dm.delete(ctx.author.id)
         await ctx.send(f"{constants.Emojis.ok_hand} {ctx.author.mention} Help DMs {state_str}!")
+
+    @tasks.loop(hours=1)
+    async def check_channel_consistency(self) -> None:
+        """
+        Check number of available help channels periodically.
+
+        If number of available help channels is incorrect, reload cog.
+        """
+        log.trace("Checking channel consistency")
+        available_help_channels = set(_channel.get_category_channels(self.available_category))
+        if len(available_help_channels) != constants.HelpChannels.max_available:
+            log.trace("Unexpected number of available help channels, reloading cog...")
+            self.bot.reload_extension("bot.exts.help_channels")
```

Comment:
Why reload the whole cog? Is it not sufficient to just call `init_available()` again?

Reply:
In such case, I think you'd need to add a lock so it doesn't interfere with `claim_channel`.